app/controllers/admin_controller.rb

```diff
@@ -13,6 +13,15 @@ class AdminController < ApplicationController
     head :bad_request
   end
 
+  def switch_locale(&action)
+    locale =
+      ENV['DEFAULT_PREFERRED_DASHBOARD_LOCALE'].presence ||
```

Comment:
Here we are making an assumption that each country has their own servers. 
Should we leave a comment for the same?

Reply:
It's not necessarily per country, it's locale-per-deployment, whatever that might be. We can later have a locale / language switcher in the dashboard, but for now we can set this for any of our deployments appropriately.